### DISCUSSION TITLE
[Push] Accept APNs Certificates `.p8`

### DISCUSSION THREAD

**Discussion by Lascorbe** [WalletConnect/walletconnect-monorepo#2006] [Developer Support]
Currently the cloud dashboard only allows to upload `.p12` certificates, but the community has moved towards using `.p8` certificates for a few years now. Are you going to allow them soon?

**Comment by HarryET** [comment]
Hi, there is a PR open to echo server now adding support. We hope to add them in the coming weeks.

**Reply by Lascorbe** [reply]
amazing thank you @HarryET!
